src/moonlink_connectors/src/pg_replicate/conversions/text.rs

```diff
@@ -351,13 +354,17 @@ impl TextFormatConverter {
                         }
                         Kind::Array(_) => {
                             // TODO: Multi-dimensional arrays not yet implemented
-                            // TODO: Need to print out the unsupported type (included in error type)
-                            Err(FromTextError::InvalidConversion())
+                            Err(FromTextError::InvalidConversion(format!(
+                                "multi-dimensional arrays"
+                            )))
                         }
-                        _ => Err(FromTextError::InvalidConversion()),
+                        other => Err(FromTextError::InvalidConversion(format!(
+                            "Array<{:?}>",
+                            other
```

Comment:
Sorry if I didn't make myself clear, if you take a look at `kind`, it doesn't include all the type information:
https://github.com/sfackler/rust-postgres/blob/e1cd6beef3a1530642a2abaf3584d6bd8ed6cd45/postgres-types/src/lib.rs#L369-L389
what I expect in the error message is full type
https://github.com/sfackler/rust-postgres/blob/e1cd6beef3a1530642a2abaf3584d6bd8ed6cd45/postgres-types/src/type_gen.rs#L6-L202